frappe/docs/user/fr/guides/app-development/trigger-event-on-deletion-of-grid-row.md

```diff
@@ -1,22 +1,45 @@
 
```

Comment:
Sorry about that, I used a direct link from discuss. I will close the pull request and move it